decidim-core/lib/decidim/translatable_resource.rb

```diff
@@ -15,5 +17,21 @@ def self.translatable_fields_list
         @translatable_fields
       end
     end
+
+    def machine_translation_new_resource
+      Decidim::MachineTranslationCreateResourceJob.perform_later(self, I18n.locale.to_s)
```

Comment:
Shouldn't we check if the MachineTranslation system is configured before invoking any related job?

Reply:
I'd invoke the job always and let the job decide what to do